docs/Server.md

```diff
@@ -631,18 +639,21 @@ fastify.setReplySerializer(function (payload, statusCode){
 })
 ```
 
-<a name="set-schema-compiler"></a>
-#### setSchemaCompiler
-Set the schema compiler for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-compiler).
+<a name="set-validator-compiler"></a>
+#### setValidatorCompiler
+Set the schema validator compiler for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-validator).
 
-<a name="set-schema-resolver"></a>
-#### setSchemaResolver
-Set the schema `$ref` resolver for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-resolver).
+<a name="set-serializer-resolver"></a>
+#### setSerializerCompiler
+Set the schema serializer compiler for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-serializer).
 
+<a name="validator-compiler"></a>
+#### validatorCompiler
+This property can be used to get the schema validator. If not set, it will be `null` until the server starts.
```

Comment:
What will it be after the server starts?

Reply:
It will be a function that, given a schema and the route's coordinates, return a compiled function (from ajv)

It could be useful to know if the user wants to use the default compiler (with all the shared schema) to generate some validations.
I used it [here](https://github.com/Eomm/fastify-schema-constraint/) for example, I will add this information 👍

Reply:
My point is that you should explain this in the documentation.